### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by benwmx**
## First Milestone:
* Set up the project using WebPack.
* Add Html structure
* Add CSS styling
* Add javascript function to load the list.

### ScreenShot
preview

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,45 @@
...
+
+## Built With
+
+- Html/Css
+- Javascript
+- Webpack
+
+## Live Demo
+
+[Live Demo Link](https://amazing-shaw-03b76e.netlify.app)
```
Comment (bellom) [inline README.md:20]: ### [Optional]

It would be nice to update your live demo link.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,45 @@
...
+
```
Comment (bellom) [inline README.md:21]: Good job setting up this README file.
If you check the [Microverse template](https://github.com/microverseinc/readme-template) you will see there is a section where you're supposed to add `Setup`.

Example: 
```
- git clone `your repo`
- cd into the folder
- git checkout `branch-name`
- run `npm i`
- run `npm start`
```

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @benwmx  👏 , 

### Awesome job so far!!, great design :+1: 

#### You have done wonderfully well setting up linter and good commit messages 🔥, However, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Required Changes! ♻️ 
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by benwmx** [comment]
hi, @bellom thank you for your suggestions, I did all the request changes.

File: README.md (hunk 1 above)
Comment (henatan99) [reply README.md:21]: - Live demo link is as it is supposed to be now. Well done! :+1: :clap:

File: README.md (hunk 2 above)
Comment (henatan99) [reply README.md:22]: - This section is added to the readme. Nice work. :clap: :clap: :clap:

**Review by henatan99 — APPROVED** [review]
Hi @benwmx , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**